rotkehlchen/data_import/importers/bittrex.py

```diff
@@ -178,8 +188,7 @@ def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) ->
         - UnsupportedCSVEntry if operation not supported
         - InputError if a column we need is missing
         """
-        consumer_fn: Callable[[dict[str, Any], BittrexFileType, str], list[AssetMovement]] | Callable[[dict[str, Any], BittrexFileType, str], Trade]  # noqa: E501
-        save_fn: Callable[[DBCursor, Trade], None] | Callable[[DBCursor, list[HistoryBaseEntry]], None]  # noqa: E501
+        consumer_fn: Callable[[dict[str, Any], BittrexFileType, str], list[AssetMovement]] | Callable[[dict[str, Any], BittrexFileType, str], list[SwapEvent]]  # noqa: E501
```

Comment:
```suggestion
        consumer_fn: Callable[[dict[str, Any], BittrexFileType, str], list[AssetMovement] | list[SwapEvent]]
```
maybe just `list[AssetMovement] | list[SwapEvent]` for the return type would work here?

Reply:
tried that, it causes mypy to raise this error `Too few arguments  [call-arg]`